.changeset/curvy-steaks-drop.md

```diff
@@ -0,0 +1,8 @@
+---
+"@kaizen/components": minor
+---
+
+Add OptionalIntlProvider to KaizenProvider.
+With OptionalIntlProvider added this way, KaizenProvider will wrap its children 
+in a StaticIntlProvider from @cultureamp/i18n-react-intl if one is not present in
+the app's context already.
```

Comment:
Whilst this is technically correct, this is to be read by our consumers, so we need to write what feature this unlocks, and not so much how it works.

We can still say "Add X" so I think in this case we should mention just the home package and then what the feature is.